### PR TITLE
[15.0][MIG] marketing_crm_partner: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT44157

@chienandalu please review!

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Please squash some administrative commits

**Comment by stefan-tecnativa** [comment]
> Please squash some administrative commits

Done, thanks for the comment

**Review by chienandalu — APPROVED** [review]
Please review @CarlosRoca13

File: marketing_crm_partner/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+To use this module, you need to:
+
+#. Go to **CRM > Leads > Create**.
+#. Fill the required fields.
+#. Go to **Extra Info > Marketing** and fill those 3 fields.
```
Comment (CarlosRoca13) [inline marketing_crm_partner/readme/USAGE.rst]:
```suggestion
#. Go to **Extra Info > Tracking** and fill those 3 fields.
```

**Review by CarlosRoca13 — APPROVED** [review]
Minimal comment

File: marketing_crm_partner/readme/USAGE.rst (hunk 1 above)
Comment (stefan-tecnativa) [reply marketing_crm_partner/readme/USAGE.rst]: Done, thanks!

**Review by chienandalu — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by francescapenso** [comment]
This should be migrate to 16.0?

**Pull request merged**